### DISCUSSION TITLE
How to use Preline with Laravel 10 + Inertia + Vue 3?

### DISCUSSION THREAD

**Discussion by HenestrosaDev** [htmlstreamofficial/preline#274] [Q&A]
Hi.

I'm following the [Laravel guide ](https://preline.co/docs/frameworks-laravel.html) from the documentation, but I'm not getting the third step right.

The entry point of my app (`resources/js/app.js`) is as follows:

```js
import { createApp, h } from "vue";
import { createInertiaApp, Link } from "@inertiajs/vue3";
import { resolvePageComponent } from "laravel-vite-plugin/inertia-helpers";
import BaseLayout from "@/Layouts/BaseLayout.vue";
import "preline/plugin";

createInertiaApp({
	progress: {
		color: "#d62",
	},
	resolve: (name) => {
		const page = resolvePageComponent(
			`./Pages/${name}.vue`,
			import.meta.glob("./Pages/**/*.vue")
		);
		page.then((module) => {
			module.default.layout = module.default.layout || BaseLayout;
		});
		return page;
	},
	setup({ el, App, props, plugin }) {
		createApp({ render: () => h(App, props) })
			.use(plugin)
			.component("Link", Link)
			.mount(el);
	},
});
```

What am I doing wrong? I've already tried using `import preline` or `import preline/preline` without success. I'm not importing anything related to Preline into my `.vue' files (which I don't think is necessary, though I could be wrong).

Thanks in advance.

**Comment by Fed3x** [comment]
Same problem :'(
